src/views/Dashboard.vue

```diff
@@ -118,21 +149,39 @@
             <!-- Proprietor / Partners -->
             <section v-if="isFirm">
               <header class="aside-header mb-3">
-                <h2 v-if="isSoleProp" data-test-id="dashboard-proprietor-subtitle">Proprietor</h2>
-                <h2 v-if="isPartnership" data-test-id="dashboard-partners-subtitle">Partners</h2>
-                <v-btn text small color="primary"
+                <h2
+                  v-if="isSoleProp"
+                  data-test-id="dashboard-proprietor-subtitle"
+                >
+                  Proprietor
+                </h2>
+                <h2
+                  v-if="isPartnership"
+                  data-test-id="dashboard-partners-subtitle"
+                >
+                  Partners
+                </h2>
+                <v-btn
+                  v-if="!isHistorical"
                   id="change-proprietor-partners-button"
+                  text
+                  small
+                  color="primary"
                   class="change-btn"
-                  v-if="!isHistorical"
                   :disabled="!isAllowed(AllowableActions.DIRECTOR_CHANGE)"
                   @click.native.stop="goToChangeFiling()"
                 >
-                  <v-icon small>mdi-pencil</v-icon>
+                  <v-icon small>
+                    mdi-pencil
+                  </v-icon>
                   <span>Change</span>
                 </v-btn>
               </header>
 
-              <div class="scrollable-container" style="max-height: 49rem">
+              <div
+                class="scrollable-container"
+                style="max-height: 49rem"
```

Comment:
This is also declared on line 85. Please create a class for this instead (or add to scrollable-container CSS?).

Reply:
Also line 219.

Reply:
Thanks.